index.bs

```diff
@@ -177,12 +178,14 @@ ErrorResponse = {
   ),
   message: text,
   stacktrace?: text,
+  *text => any
```

Comment:
This is an addition to `ErrorResponse`. Does "respond with an error" also need to be updated to populate this, or how does it fit together?

Reply:
Let's handle the extensible discussion in a seperate PR.

Reply:
This is the extensibility stuff we were discussing elsewhere. I suggest we don't block this PR on it, but clean up all the productions that ought to be extensible later.

Reply:
When posting this comment I didn't understand what `*text => any` was and thought it was a new `text` field. But yeah, let's discuss in https://github.com/w3c/webdriver-bidi/issues/79.